source/includes/_contributing.mdown

```diff
@@ -1,32 +1,41 @@
-# Getting help
+# Contributing
 
-## Discord
+Want to hack on Origin? Awesome!
 
-Our team collaboration is done in public and our company Discord is open to all. If you have questions or need help getting started, our [Discord #engineering channel](https://www.originprotocol.com/discord) is a great place to get assistance from our team of engineers and developers.
+Origin is an Open Source project and we welcome contributions of all sorts. There are many ways to help, from reporting issues, contributing code, and helping us improve our community.
 
-## Email
+There are four main repositories:
 
-You can also reach us by email at [support@originprotocol.com](mailto:support@originprotocol.com).
+- [Origin.js](https://github.com/OriginProtocol/origin-js) - solidity contracts and the Origin JS API
+- [Demo DApp](https://github.com/OriginProtocol/demo-dapp) - example react powered front end 
+- [Bridge Server](https://github.com/OriginProtocol/bridge-server) - python indexing and search server
+- [Identity Playground](https://github.com/OriginProtocol/identity-playground) - implementation of ERC 725/735 for blockchain identity
 
-# Contributing
+### Dive Right In
 
-Want to hack on Origin? Awesome!
+If you're ready to start hacking on Origin right now and you just need an issue to focus on, check out [this list of good first issues](https://github.com/orgs/OriginProtocol/projects/2?card_filter_query=label%3A%22good+first+issue%22+no%3Aassignee).
```

Comment:
Only the four product-related repos are guaranteed to have their issues in the project board. So many of the "good first issues" in the docs repo and company-website repo will be found by searching [here](https://github.com/OriginProtocol).

Reply:
Actually, the search experience at that top level of the GitHub organization is not very intuitive. This is another of several reasons to add the company-website repo to the project board.